framework/13-inch/common/default.nix

```diff
@@ -6,9 +6,7 @@
 
   # Fix TRRS headphones missing a mic
   # https://community.frame.work/t/headset-microphone-on-linux/12387/3
-  #
-  # This is temporary until a kernel patch is submitted
-  boot.extraModprobeConfig = ''
+  boot.extraModprobeConfig = lib.mkIf (lib.versionOlder pkgs.linux.version "6.6.8") ''
```

Comment:
pkgs also needs to be passed in the module header of this file.

Reply:
Done, thanks